client/src/components/Accordion/Accordion.js

```diff
@@ -122,10 +122,10 @@ const [AdvancedAccess, setAccess] = React.useState(false);
   axios.get('http://localhost:2222/users', { params: { email: safeEmail } } )
   .then(res => {
     //Check for the users id in the signed up users
-    setUserID(res.data._id);
+    setUserID(res.data[0]._id);
```

Comment:
Curious to how/why this changed @benhsm

Reply:
Adding what I said on slack to clarify this for anyone reading this here who isn't in the slack workspace or project channel:

> The prior behaviour for the GET /users endpoint was to return a single user object if provided with an email query parameter, and otherwise to simply return all users as a list. This seemed inconsistent to me. I suppose emails are unique in the User model, so a query by email would only ever return one object.
>
>In preparation for implementing the admin UI, I thought it would be good to make it possible to use filter criteria to retrieve users (e.g. find all the volunteers who have horse-leading skills) and conceptually make the /users endpoint more about dealing with lists of users, especially since the /me and /users/:id endpoints deal with singular users.